### ISSUE TITLE
wxMaxima failing to evaluate any cells

### ISSUE THREAD

**Issue by samfearn** [wxMaxima-developers/wxmaxima#1761]
I've recently re-installed maxima on an M1Pro Mac, running Ventura 13.2.1, via MacPorts, following the instructions from the 5.46.0-macOS version of Maxima from Sourceforge.

wxMaxima launches fine, and the GUI is responsive, but I'm not able to evaluate any cells. When I try, wxMaxima just shows "1 cells in evaluation queue" in the status bar, and "Reading Maxima output" in the right hand side of the status bar.

Here's the debug messages, in case it's helpful:

`12:24:22: wxMaxima version 22.04.0
12:24:22: OS: Mac OS X (Darwin 22.3.0 arm64) Version 13.2
12:24:22: Running on Mac OS
12:24:22: wxWidgets 3.0.5
12:24:22: Translations are read from /Applications/MacPorts/wxMaxima.app/Contents/Resources/locale.
12:24:22: Reading the config from the default location.
12:24:22: Reading the glyph renderability cache
12:24:22: Font 0;10;70;90;90;0;Monaco;49 should render the glyphs "αβγδεζηθικλνξπρστυφχψωΓΔΘΛΞΠΣΦΨΩ½²³√ⅈⅇℏ∈∃∄⇒∞∅▶▸⋀⋁⊻⊼⊽⇔±¬⋃⋂⊆⊂⊈⊄ħĦ∂∇∫≅∝≠≤≥≪≫≣∑∏∥⟂↝→⟶∎üØ" correctly.
12:24:22: Font 0;13;70;90;90;0;.AppleSystemUIFont;49 should render the glyphs "½²³√ⅈⅇℏ∈∃∄⇒∞∅▶▸⋀⋁⊻⊼⊽⇔±¬⋃⋂⊆⊂⊈⊄ħĦ∂∇∫≅∝≠≤≥≪≫≣∑∏∥⟂↝→⟶∎" correctly.
12:24:22: Font 0;10;70;90;90;0;Monaco;49 cannot render the glyphs "".
12:24:22: Font 0;13;70;90;90;0;.AppleSystemUIFont;49 cannot render the glyphs "".
12:24:22: Reading the glyph renderability cache
12:24:22: Font 0;10;70;90;90;0;Monaco;49 should render the glyphs "αβγδεζηθικλνξπρστυφχψωΓΔΘΛΞΠΣΦΨΩ½²³√ⅈⅇℏ∈∃∄⇒∞∅▶▸⋀⋁⊻⊼⊽⇔±¬⋃⋂⊆⊂⊈⊄ħĦ∂∇∫≅∝≠≤≥≪≫≣∑∏∥⟂↝→⟶∎üØ" correctly.
12:24:22: Font 0;13;70;90;90;0;.AppleSystemUIFont;49 should render the glyphs "½²³√ⅈⅇℏ∈∃∄⇒∞∅▶▸⋀⋁⊻⊼⊽⇔±¬⋃⋂⊆⊂⊈⊄ħĦ∂∇∫≅∝≠≤≥≪≫≣∑∏∥⟂↝→⟶∎" correctly.
12:24:22: Font 0;10;70;90;90;0;Monaco;49 cannot render the glyphs "".
12:24:22: Font 0;13;70;90;90;0;.AppleSystemUIFont;49 cannot render the glyphs "".
12:24:22: FontCache Raw Miss: 10.00pt  ---- "Monaco" fam:70 enc:0
12:24:22: FontCache Resolved: 10.00pt  ---- "Monaco" fam:70 enc:49
12:24:22: FontCache Raw Miss: 10.00pt  BI-- "Monaco" fam:70 enc:0
12:24:22: FontCache Resolved: 10.00pt  BI-- "Monaco" fam:70 enc:49
12:24:22: Gnuplot found at: /opt/local/bin/gnuplot
12:24:22: Selected language: en_GB (58)
12:24:22: LANG=en_GB.UTF-8
12:24:22: Trying to start the socket a maxima on the local machine can connect to on port 4010
12:24:22: Closing the socket maxima could connect to!
12:24:22: Trying to start the socket a maxima on the local machine can connect to on port 4011
12:24:22: Server started
12:24:22: Running maxima as: "/opt/local/bin/maxima"  -s 4011 
12:24:23: Setting a few prerequisites for wxMaxima
12:24:23: Sending maxima the info how to express 2d maths as XML
12:24:23: Reading the Lisp part of wxMaxima from the included header file.
12:24:23: Setting gnuplot_binary to /opt/local/bin/gnuplot
12:24:23: Updating maxima's configuration
12:24:23: Sending configuration data to maxima.
12:24:23: Received maxima's first prompt: pid=51243↲Maxima 5.46.0 https://maxima.sourceforge.io↲using Lisp SBCL 2.2.3↲Distributed under the GNU Public License. See the file COPYING.↲Dedicated to the memory of William Schelter.↲The function bug_report() provides bug reporting information.↲(18023896641) 
12:24:23: Maxima's PID is 51243
12:24:23: Ready for user input
12:24:23: Maxima is ready for input.
12:24:31: FontCache Raw Miss:  4.00pt  BI-- "Monaco" fam:70 enc:0
12:24:31: FontCache Resolved:  4.00pt  BI-- "Monaco" fam:70 enc:49
12:24:31: FontCache Raw Miss: 13.20pt  BI-- "Monaco" fam:70 enc:0
12:24:31: FontCache Resolved: 13.00pt  BI-- "Monaco" fam:70 enc:49
12:24:31: FontCache Raw Miss: 12.00pt  ---- "Monaco" fam:70 enc:0
12:24:31: FontCache Resolved: 12.00pt  ---- "Monaco" fam:70 enc:49
12:24:31: FontCache Raw Miss: 13.20pt  ---- "Monaco" fam:70 enc:0
12:24:31: FontCache Resolved: 13.00pt  ---- "Monaco" fam:70 enc:49`

Also, xMaxima works fine. It just seems like wxMaxima is failing to properly talk to the maxima binary?

Any help appreciated.

Thanks,

Sam

**Comment by daute** [comment]
That seems to be a Maxima (or wxMathML.lisp) issue.
Maxima 5.43.2 (included in Ubuntu 20.04) does the XML conversion, when one execute it with our wxMathML.lisp file:
`maxima --init-lisp=src/wxMathML.lisp`

A current Maxima development version emits errors and does not correctly convert the output to XML.
Very strange...

Best regards, Wolfgang

**Comment by gunterkoenigsmann** [comment]
@daute: Have you the means to test if the issue is still present? With the current development version of maxima I seem to have a running development version of wxMaxima on my system.

**Comment by daute** [comment]
No, that issue seems to be solved using the current Git version.
I compiled yesterday evening a Windows version to see, if bug  #1768 is solved (that issue is still a problem; I spent some time to find a solution, but was not able to solve it), but this problem seems to be okay...

**Comment by mlomonaco** [comment]
On version 23.04 (MacOS via brew) I get the same symptoms everytime I try to load a package via `load()` command. On terminal maxima 5.46 it works fine.

**Comment by gunterkoenigsmann** [comment]
I believe this has to do with the quarantine stuff on the Mac that disallows Maxima to connect to anything unles you use a terminal to mark it harmless.

**Comment by gunterkoenigsmann** [comment]
Now our homepage tells how to deal with this.
